front/admin_ui/app.py

```diff
@@ -175,6 +179,46 @@ def get_dataset_status(token, dataset):
                 cached_responses_table: gr.update(value=None),
                 jobs_table: gr.update(value=None)
             }
+    
+    def get_backfill_plan(token, dataset):
+        headers = {"Authorization": f"Bearer {token}"}
+        response = requests.get(f"{DSS_ENDPOINT}/admin/dataset-state?dataset={dataset}", headers=headers, timeout=60)
+        if response.status_code != 200:
+            return {
+                backfill_message: gr.update(value=f"❌ Failed to get backfill plan for {dataset} (error {response.status_code})", visible=True),
+                backfill_plan_table: gr.update(value=None,visible=False),
+                backfill_execute_button: gr.update( visible=False),
+                backfill_execute_error: gr.update( visible=False)
+            }
+        dataset_state = response.json()
+        tasks_df = pd.DataFrame(dataset_state["plan"])
+        has_tasks = len(tasks_df) > 0
+        return {
+            backfill_message: gr.update(
+                value="""### Backfill plan
+
+The cache is outdated or in an incoherent state. Here is the plan to backfill the cache."""
+            ,visible=has_tasks),
+            backfill_plan_table: gr.update(value=tasks_df,visible=has_tasks),
```

Comment:
```suggestion
            backfill_plan_table: gr.update(value=tasks_df, visible=has_tasks),
```

Reply:
If someone wants to create an issue and assign themself, please do!

Reply:
Sure no problem, we can handle it in another PR/issue